app/client/templates/elements/compileContract.js

```diff
@@ -142,7 +142,15 @@ Template['elements_compileContract'].onRendered(function() {
     this.aceEditor.$blockScrolling = Infinity;
     this.aceEditor.focus();
 
-    var defaultCode = localStorage['contractSource'] || "contract MyContract {\n    /* Constructor */\n    function MyContract() {\n \n    }\n}";
+    var defaultCode = localStorage['contractSource'] || 
+`pragma solidity ^0.4.2;
```

Comment:
I know, it's rather counter-intuitive, but now it's mandatory solidity code anyway (see opening post ☝️).